### DISCUSSION TITLE
submit error???????????

### DISCUSSION THREAD

**Discussion by RobinHan24** [PaddlePaddle/PaddleSpatial#86] [BaiduKDDCup2022] [answered]
I have run the test code successful in my local environment using the script evaluation.py and there is no any word like "sdwpf_baidukddcup2022_full.csv" in my submit code but my submit result :[Errno 2] No such file or directory: '/tmp/tmpaxukiqum//home/aistudio/eval_prog/kddcup22-sdwpf-evaluation/data/sdwpf_baidukddcup2022_full.csv'". Please help me, thanks a lot.

**Comment by ramber1836** [comment, marked as answer]
> I have run the test code successful in my local environment using the script evaluation.py and there is no any word like "sdwpf_baidukddcup2022_full.csv" in my submit code but my submit result :[Errno 2] No such file or directory: '/tmp/tmpaxukiqum//home/aistudio/eval_prog/kddcup22-sdwpf-evaluation/data/sdwpf_baidukddcup2022_full.csv'". Please help me, thanks a lot.

You can provide your team name and we will check the backend log.
Due to the hard workload, the reply might not be as soon as you expected. 
Thanks for your understanding.
